rewrite-java/src/main/java/org/openrewrite/java/RemoveImport.java

```diff
@@ -172,6 +175,10 @@ public RemoveImport(String type, boolean force) {
         return j;
     }
 
+    private long countTrailingLinebreaks(Space space) {
+        return space.getLastWhitespace().chars().filter(s -> s == '\n').count();
```

Comment:
I am a bit unsure why this doesn't fail the build as `String#chars()` was added in Java 9 and we should have the build configured to require Java 8.

Any ideas, @timtebeek ?

Reply:
Hmm... Apparently `CharSequence#chars()` was already added as a default method in Java 8. What was added in Java 9 was just the `String#chars()` override for better performance.